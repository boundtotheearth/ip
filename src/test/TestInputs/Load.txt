DEADLINE|false|return book|2021-10-11
DEADLINE|false|aaa|2020-10-11
EVENT|false|go home|2020-10-10
TODO|false|better gui
TODO|false|extract command class
