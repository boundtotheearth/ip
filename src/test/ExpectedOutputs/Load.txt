1. [D][✘] return book (at: 11 Oct 2021)
2. [D][✘] aaa (at: 11 Oct 2020)
3. [E][✘] go home (at: 10 Oct 2020)
4. [T][✘] better gui
5. [T][✘] extract command class